api/apiroutes.go

```diff
@@ -1062,6 +1063,18 @@ func (c *appContext) getAddressTransactionsRaw(w http.ResponseWriter, r *http.Re
 	writeJSON(w, txs, c.getIndentQuery(r))
 }
 
+// TicketPrices is the handler for charts.
```

Comment:
What does it show to the user?  It is not just the handler for the charts, although it happens to be used that way.  It handles returns ticket prices for all of the stake diff intervals.